grpc/src/main/java/com/linecorp/armeria/client/grpc/GrpcClientBuilder.java

```diff
@@ -399,6 +411,12 @@ public <T> T build(Class<T> clientType) {
 
         final Object client;
         final ClientOptions options = buildOptions();
+        if (options.get(GrpcClientOptions.UNSAFE_WRAP_RESPONSE_BUFFERS) && options.get(USE_METHOD_MARSHALLER)) {
```

Comment:
Could you move logic into `enableUnsafeWrapResponseBuffers` and `useMethodMarshaller` methods so that the exception is thrown early when the property is set?

Reply:
Okay, I'll move logic to `setXX` method.

Reply:
Applied feedbacks, Thanks.